appengine/firebase-tictactoe/README.md

```diff
@@ -29,26 +23,36 @@ for realtime notifications when the board changes.
 [fb-overview]: https://console.firebase.google.com/project/_/overview
 [maven]: https://maven.apache.org
 
+## Setup
+
+* If you haven't already, Download and initialize the [Cloud SDK](https://cloud.google.com/sdk/)
+```
+    gcloud init
+```
+* If you haven't already, Create an App Engine app within the current Google Cloud Project
+```
+    gcloud app create
+```
+* If you haven't already, setup
+[Application Default Credentials](https://developers.google.com/identity/protocols/application-default-credentials)
+to run your application locally:
+Download [service account credentials][creds] and set the `GOOGLE_APPLICATION_CREDENTIALS`
+environment variable to its path:
+```
+   export GOOGLE_APPLICATION_CREDENTIALS=/path/to/your/credentials.json
```

Comment:
Should we have them set GOOGLE_CLOUD_PROJECT?  Look at gcloud config to verify their project?

Reply:
given we still ask users to use `gcloud init` : added a note there that this sets the project.